client/src/index.jsx

```diff
@@ -1,8 +1,38 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import axios from 'axios';
 
-const App = () => (
-	<h1>Hello from React!</h1>
-);
+class App extends React.Component {
+	constructor(props) {
+		super(props);
+		this.getSimilarListings = this.getSimilarListings.bind(this);
```

Comment:
You may not need to bind the getSimilarListings function since it will only be invoked within the App component.

Reply:
Good point! I will refactor.